src/keyboard-toolbar.android.ts

```diff
@@ -26,12 +25,12 @@ export class Toolbar extends ToolbarBase {
 
 	constructor() {
 		super();
-
 		this.verticalAlignment = "top"; // weird but true
+		this.isShowingKeyboard = false;
```

Comment:
The name `isShowingKeyboard` is a bit ambiguous because it can be

- Can keyboard show, or
- Is keyboard is opened <-- I think this is what you mean

How about change it to `isKeyboardOpened`?